extensions/amp-fx-scroll-sync/0.1/scroll-sync-scale-effect.js

```diff
@@ -25,6 +25,9 @@ export class ScrollSyncScaleEffect extends ScrollSyncEffect {
     this.endScale_ = config['end-scale'];
     this.scaleOrigin_ = config['scale-origin'];
     this.layoutBox_ = null;
+    if (this.scaleOrigin_) {
+      setStyle(this.element, 'transform-origin', this.scaleOrigin_);
```

Comment:
`scaleOrigin` is not needed again, I can omit setting it as a property, but after [this](https://github.com/ampproject/amphtml/pull/4584/files#diff-44bf2ab7edc5fc9afedbb5a67bf6f093R42) is merged.